src/main/java/com/codersbay/MyStack.java

```diff
@@ -0,0 +1,43 @@
+package com.codersbay;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyStack<T> {
+    private final List<T> content = new ArrayList<T>();
```

Comment:
🔝 final verwenden ist immer eine gute Idee 👍

Reply:
hat sich "richtig" angespürt, ich kann aber ned konkret sagen, warum ich's genommen hab - warum ist das im Falle einer Liste in einer Klasse eine gute Idee?

Reply:
final ist für jede variable der kein neuer Wert zugewiesen werden darf eine gute idee - unabhängig vom datentyp, es erleichtert es dem compiler und dem programmier da man weiß dass sich der wert dieser variable nicht mehr komplett ändert. (der liste dürfen zwar elemente hinzufgefügt werden aber der variable darf nicht eine neue liste zugewiesen werden)